$OpenBSD: patch-src_libwaitress_waitress_h,v 1.1 2012/11/30 15:52:56 dcoppa Exp $

commit a55511ea75003f5ca79a25f6e64eadf91b6dfe9f
Author: Lars-Dominik Braun <lars@6xq.net>
Date:   Fri Nov 30 15:10:36 2012 +0100

    waitress: Fix chunked decoder

    Fix "Get stations... Network error: Invalid encoded data."

--- src/libwaitress/waitress.h.orig	Fri Nov 30 16:45:05 2012
+++ src/libwaitress/waitress.h	Fri Nov 30 16:45:30 2012
@@ -106,6 +106,7 @@ typedef struct {
 		WaitressReturn_t readWriteRet;
 
 		size_t contentLength, contentReceived, chunkSize;
+		enum {CHUNKSIZE = 0, DATA = 1} chunkedState;
 
 		char *buf;
 		/* first argument is WaitressHandle_t, but that's not defined yet */
